Library/PTPusher.m

```diff
@@ -274,6 +274,20 @@ - (void)subscribeAll
   }
 }
 
+- (void)unsubscribeAllChannels
+{
+  if (channels && channels.count) {
+	NSMutableArray *channelsToUnsubscribe = [[NSMutableArray alloc] init];
+	  for (PTPusherChannel *channel in [channels allValues]) {
+		[channelsToUnsubscribe addObject:channel];
```

Comment:
No problem, you'd need to check the method `- (void)__unsubscribeFromChannel:(PTPusherChannel *)channel` to figure that out.